<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模拟进程并发执行</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_a">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_a_suspend">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>20</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_a_resume">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>90</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_b">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_b_suspend">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>20</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_b_resume">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>90</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_c">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_c_suspend">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>20</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_c_resume">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>90</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labeli_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
